### DISCUSSION TITLE
Autorestart/poll from slaves

### DISCUSSION THREAD

**Discussion by knofte** [oetiker/SmokePing#362] [Ideas]
When updating config on master.
Let slaves poll f.ex every hour if there's a new config on master.

When you have 5+ slaves, it's a bit of a hassle to login to each of them to restart just because you added 1 more host to monitor on master.

**Comment by svenvg93** [comment]
Hi @knofte, 

We used ansible to restart all the slaves services after we changed the config. 

https://docs.ansible.com/ansible/latest/collections/ansible/builtin/service_module.html
